src/themes/default/components/core/blocks/Reviews/Reviews.vue

```diff
@@ -126,7 +126,7 @@ export default {
   },
   props: {
     productId: {
-      type: Number,
+      type: null,
```

Comment:
Just one more here - you can also do here Number or String in vue prop :)

Reply:
Updated. Please check. 
Updated changelog too